src/instruments/src/PFD/AltitudeIndicator.jsx

```diff
@@ -168,7 +168,7 @@ const AltimeterIndicator = ({ mode, alt }) => {
 
     if (mode === 'STD') {
         return (
-            <g id="STDAltimeterModeGroup" className={(alt < transAlt && transAlt !== 0) ? 'BlinkInfinite' : ''}>
+            <g id="STDAltimeterModeGroup" className={(phase !== 7 && phase > 3 && transAlt > alt && transAlt !== 0) ? 'BlinkInfinite' : ''}>
```

Comment:
What is the `phase !== 7` supposed to do?

Reply:
Done phase.
idk which pilot did that but after landing, if change to std, will flashing. so to prevent that.

Reply:
I assume inhibit when in Done phase, but at the same time the code inhibits unless in or past Cruise.

Reply:
I don't understand. If the real aircraft still flashes after landing, then it would be correct without the `phase !== 7`.

Reply:
just wait answer from pilot that it inhibit on ground :D

Reply:
As I said on your previos PR, unless they specifically tested this scenario on a real aircraft, it's not going to be reliable info,
since no pilot would realistically ever encounter this naturally. I would rather remove this from the PR and get it merged, than to wait for someone to test it in a real aircraft. If it turns out to be true, it can always be added later.

Reply:
Ok